addons/project_purchase/static/src/product_catalog/kanban_record.js

```diff
@@ -0,0 +1,16 @@
+/** @odoo-module **/
+import { patch } from "@web/core/utils/patch";
+
+import { ProductCatalogKanbanRecord } from "@product/product_catalog/kanban_record";
+
+patch(ProductCatalogKanbanRecord.prototype, {
+    _updateQuantityAndGetPrice() {
```

Comment:
@chevalierv Can we add a hook to alter the params given to the rpc? 

As it is the case in master (and also 17.2)?

IMO, it would be easier for the custom module to also alter that hook if needed. Otherwise, we will have to copy/paste the content of `_updateQuantityAndGetPrice` and add the additional parameter(s).

What do you think? 🙂

Reply:
Hello @xavierbol 

I don't see why this couldn't be done in 17.0 so LGTM :+1:

Reply:
@chevalierv cool thanks. 💪🏼